common/pxc-db/values.yaml

```diff
@@ -0,0 +1,389 @@
+# Default values for percona-xtradb-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+linkerd:
+  enabled: true
+
+owner_info:
+  enabled: false
+
+name: null
+
+# NOTE: don't use internal TLS by default, linkerd mesh is preferred
+tls:
+  enabled: false
+
+unsafeFlags: {}
+#unsafeFlags:
+#  pxcSize: false
+#  proxySize: false
+#  backupIfUnhealthy: false
+
+character_set_server: "utf8mb4"
+collation_server: "utf8mb4_0900_ai_ci"
+
+initdb_secret: null
+
+databases: {}
+users:
+  backup:
+    name: backup
+    password: null
+    limits:
+      max_user_connections: 4
+    grants:
+      - ALL PRIVILEGES ON *.*
+#  example:
+#    name: example1 # This looks repetitive, but the point is that they key is the name
+#                   # you refer to in your charts, while the field 'name' is the actual name
+#                   # used as credentials. It should be possible to change the latter,
+#                   # without having to change the first.
+#    password: null # Causes users not be be created, and even maybe to get locked
+#    auth_plugin: 'mysql_native_password'
+#    grants:
+#    - ALL ON example.*
+
+system_users:
+  root:
+    password: null
+  xtrabackup:
+    password: null
+  monitor:
+    password: null
+  proxyadmin:
+    password: null
+  operator:
+    password: null
+  replication:
+    password: null
+
+ignoreAnnotations: []
+ignoreLabels:
+  - ccloud/service
+  - ccloud/support-group
+
+annotations: {}
+
+pause: false
+
+initContainer:
+  image:
+    name: percona/percona-xtradb-cluster-operator
+    tag: 1.15.0
+    override: null
+  resources:
+    requests:
+      memory: 100M
+      cpu: 100m
+    limits:
+      memory: 200M
+      cpu: 200m
+
+pxc:
+  size: 3
+  image:
+    name: percona/percona-xtradb-cluster
+    tag: 8.0.37-29.1
+    override: null
+  imagePullPolicy: IfNotPresent
+  annotations: {}
+  labels:
+    system: openstack
+    type: database
+  readinessProbes:
+    initialDelaySeconds: 15
+    timeoutSeconds: 15
+    periodSeconds: 30
+    successThreshold: 1
+    failureThreshold: 5
+  livenessProbes:
+    initialDelaySeconds: 300
+    timeoutSeconds: 5
+    periodSeconds: 10
+    successThreshold: 1
+    failureThreshold: 3
+  containerSecurityContext:
+    privileged: false
+  podSecurityContext:
+    runAsUser: 1001
+    runAsGroup: 1001
+    fsGroup: 1001
+    supplementalGroups: [1001]
+    fsGroupChangePolicy: "Always"
+  expose:
+    enabled: false
+    type: ClusterIP
+    externalTrafficPolicy: Cluster
+    internalTrafficPolicy: Cluster
+    annotations:
+      config.linkerd.io/opaque-ports: "3306,3307,3009,4444,4567,4568,33060,33062"
+  autoRecovery: true
+  configuration: |
+    [mysqld]
+    wsrep_provider_options          = "gcache.size=1G; gcache.recover=yes"
+    wsrep_slave_threads             = 4
+    wsrep_retry_autocommit          = 3
+    binlog_format                   = ROW
+    max_connections                 = 1024
```

Comment:
jfyi: we have higher limits for nova/neutron at least in large regions.

Reply:
Moved all these options to hash `pxc.configuration.options`. Now these options can be overwritten individually.

```
pxc:
  configuration:
    options:
      max_connections: 2048
      innodb_buffer_pool_size: 2048M
```